class.jetpack-gutenberg.php

```diff
@@ -6,16 +6,27 @@
  * @package Jetpack
  */
 
+/**
+ * Helper function to register a Jetpack Gutenberg extension
+ *
+ * @param string $slug Slug of the extension.
+ *
+ * @return boolean True if the extension is registered and False if it is not whitelisted
+ */
+function jetpack_register_gutenberg_extension( $slug ) {
+	return Jetpack_Gutenberg::register_extension( $slug );
+}
+
 /**
  * Helper function to register a Jetpack Gutenberg block
  *
+ * @deprecated 7.0.0 Use jetpack_activate_gutenberg_extension() + register_block_type() instead
```

Comment:
Did you mean to say "Use jetpack_register_gutenberg_extension()", since there is no "jetpack_activate_gutenberg_extension()"?

Reply:
Nice catch! I initially used `jetpack_activate_gutenberg_extension` as the function name but decided later to rename it to `jetpack_register_gutenberg_extension` and I forgot to replace it here 😉. Addressed in 98fad77.